rust/hyperlane-base/src/settings/base.rs

```diff
@@ -157,7 +157,7 @@ impl Settings {
     }
 
     /// Try to get a CachingMailbox
-    async fn build_caching_mailbox(
+    pub async fn build_caching_mailbox(
```

Comment:
Where is this now getting used?

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/92bc8915692a869730834809f0638cb6101d43f0/rust/agents/validator/src/validator.rs#L59
in the validator?